start_lightly_worker.sh

```diff
@@ -0,0 +1,3 @@
+#! /usr/bin/bash
+docker run --shm-size="1024m" --gpus all --rm -it -e LIGHTLY_TOKEN={MY_LIGHTLY_TOKEN} \ 
+    lightly/worker:latest token=your_awesome_token worker.worker_id={MY_WORKER_ID}
```

Comment:
Could you drop the `token=your_awesome_token` part? The token is already passed with `LIGHTLY_TOKEN={MY_LIGHTLY_TOKEN}`.